README.md

```diff
@@ -3,10 +3,18 @@
 ## Getting Started
 1. Set the correct Node version via `nvm use`
 2. Start the Firebase Emulators via `npm run dev`
-3. In another Terminal, start the React development server by running `npm start` within the `hosting` directory
-   - In order to authenticate locally, you will need to use `127.0.0.1` rather than `localhost`
+3. In another Terminal:
+   1. Navigate into the `hosting` directory
+   2. Run `npm i`
```

Comment:
How come you mention `npm i` here, but not for the root directory?

Reply:
There's no dependencies in the root directory - the Firebase CLI is triggered via `npx` so doesn't need to be pre-installed via `npm i`